ddtrace/internal/_encoding.pyx

```diff
@@ -548,6 +551,11 @@ cdef class MsgpackEncoderV03(MsgpackEncoderBase):
                     ret = pack_bytes(&self.pk, _ORIGIN_KEY, _ORIGIN_KEY_LEN)
                     if ret == 0:
                         ret = pack_bytes(&self.pk, dd_origin, strlen(dd_origin))
+                if i == 0:
+                    ret = pack_text(&self.pk, <str> "language")
+                    if ret != 0: return ret
+                    ret = pack_text(&self.pk, <str> "python")
```

Comment:
we will want to use a `const char*` here like for origin (L79) for both the key and the value.

`pack_text` has some conversion that may happen to get bytes and then append, where if we start with a properly formed `const char*`/bytes anyways, they can just be appended to the payload with minimal overhead (we just need to mem copy the bytes).